spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -191,25 +191,27 @@ def add_beta_groups_to_build(build_id: nil, beta_group_ids: [])
           test_flight_request_client.post("builds/#{build_id}/relationships/betaGroups", body)
         end
 
-        def create_beta_group(app_id: nil, group_name: nil, public_link_enabled: false, public_link_limit: 10_000, public_link_limit_enabled: false)
+        def create_beta_group(app_id: nil, group_name: nil, isInternalGroup: false, public_link_enabled: false, public_link_limit: 10_000, public_link_limit_enabled: false)
           body = {
             data: {
               attributes: {
                 name: group_name,
+                isInternalGroup: isInternalGroup,
+                hasAccessToAllBuilds: isInternalGroup ? true : false,
```

Comment:
Is it also an undocumented attribute? If that's the case, it could be:
`hasAccessToAllBuilds: is_internal_group,` when parameter is renamed to snake_case

Reply:
Here is the official standard. I got it by grabbing the bag.